src/component/Header/Navigation.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Navigation() {
+  return (
+    <ul>
+      <Link to="/cocktails">
+        <p className="links">Home Page</p>
+      </Link>
```

Comment:
it's only one link, so it shouldn't be called linkS + try the following:

    <Link className="link" to="/cocktails">
      Home page
    </Link>